public/components/metrics/top_menu/metrics_export_panel.tsx

```diff
@@ -75,10 +76,10 @@ export const MetricsExportPanel = ({
 
   useEffect(() => {
     fetchAllvisualizationsById();
-  }, []);
+  }, [fetchAllvisualizationsById]);
```

Comment:
wouldn't this go into a loop?
1. component mount
1. `fetchAllvisualizationsById` is called
2. `fetchAllvisualizationsById` sets visualizations metadata state
3. rerender
4. a new `fetchAllvisualizationsById` function is created
5. step 2 repeats